### PR TITLE
Added addresses for important functions without eu0 addresses

### PR DESCRIPTION
**Pull request by L5050**
Gave addresses to the following functions:
* pouchSetAttack
* pouchGetAttack
* pouchAddAttack
* pouchSetHp
* pouchSetMaxHp
* pouchGetMaxHp
* pouchSetCoin
* pouchGetCoin
* pouchAddCoin
because they didn't have eu0 addresses, I also gave them US0, US2, and JP0 addresses

### CODE REVIEW

**Comment by L5050** [comment]
this isnt ready to be merged because "others.h" needs to be merged with other files and im not sure where to put the values in it
also not sure why the ctx check failed

**Comment by L5050** [comment]
ready for you to check things out whenever you've got a chance
again, I'm not sure where to put the values in others.h

**Comment by SeekyCt** [comment]
First problem is that CI is failing, if you look at the [log of one of the failed builds](https://github.com/SeekyCt/spm-headers/actions/runs/10101109801/job/27933945604?pr=18) you can see `namespace spm::others` is causing an error when trying to build in C mode, which can be fixed by using the CPP_WRAPPER macro like other files in others.h

**Comment by SeekyCt** [comment]
As for where the contents of others.h actually belong, I'll try look into that

**Comment by SeekyCt** [comment]
> As for where the contents of others.h actually belong, I'll try look into that

Pretty sure they're part of `npc_dimeen_l.c`

File: include/spm/evtmgr.h [hunk 1]
```diff
@@ -139,6 +139,7 @@ DECOMP_STATIC(s32 evtmgr_runMainF)
 */
 EvtWork * evtGetWork();
 
+void make_jump_table(EvtEntry* entry);
```
Comment (SeekyCt) [inline include/spm/evtmgr.h]: This should use the DECOMP_STATIC macro

File: include/spm/others.h [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+namespace spm::others {
+
+extern "C"
+{
+void superDimentioHeadDefenses();
+void superDimentioBodyDefenses();
+}
+  }
```
Comment (SeekyCt) [inline include/spm/others.h]: See previous replies

File: linker/spm.jp0.lst [hunk 3]
```diff
@@ -3,6 +3,9 @@
 800030c8:firstRel
 800030cc:lastRel
 
+
+803F0E88:superDimentioHeadDefenses
+803F0EA0:superDimentioBodyDefenses
```
Comment (SeekyCt) [inline linker/spm.jp0.lst]: These should be grouped in with the rest of the hard mode symbols (same in us0, us2)

**Review by SeekyCt — CHANGES_REQUESTED** [review]
Couple small things, pretty much ready to merge after that

**Comment by L5050** [comment]
I dont understand why evt_npc_get_hp is causing the builds to fail, but otherwise everything is resolved

**Comment by SeekyCt** [comment]
> I dont understand why evt_npc_get_hp is causing the builds to fail, but otherwise everything is resolved

I think it's because mwcc doesn't like you putting a semi colon at the end (the macro includes one)

File: include/spm/evt_npc.h [hunk 4]
```diff
@@ -309,5 +309,3 @@ UNKNOWN_FUNCTION(func_8010c2e0);
 UNKNOWN_FUNCTION(func_8010c35c);
 UNKNOWN_FUNCTION(func_8010c3e8);
 UNKNOWN_FUNCTION(func_8010c440);
-
-CPP_WRAPPER_END()
```
Comment (SeekyCt) [inline include/spm/evt_npc.h]: I don't think you wanted to remove this

Comment (L5050) [reply include/spm/evt_npc.h]: thank you lol

Comment (SeekyCt) [reply include/spm/evt_npc.h]: The changes to evt_cmd.h can probably be reverted now

**Comment by L5050** [comment]
everything should be resolved now :)

File: include/spm/evt_npc.h (hunk 4 above)
Comment (L5050) [reply include/spm/evt_npc.h]: done, sorry I misread originally lol

File: linker/spm.eu0.lst [hunk 5]
```diff
@@ -6,7 +6,8 @@
 // more
 
 /* Dol */
-
+8045B9E0:superDimentioHeadDefenses
```
Comment (SeekyCt) [inline linker/spm.eu0.lst]: These should be moved under npc_dimeen_l.c

**Review by SeekyCt — CHANGES_REQUESTED** [review]
Happy to merge after this

File: linker/spm.eu0.lst (hunk 5 above)
Comment (L5050) [reply linker/spm.eu0.lst]: all done!

**Review by SeekyCt — APPROVED** [review]

**Pull request merged**